src/vellum/workflows/inputs/base.py

```diff
@@ -41,7 +41,16 @@ def __getattribute__(cls, name: str) -> Any:
             else:
                 return WorkflowInputReference(name=name, types=types, instance=instance, inputs_class=cls)
 
-        return super().__getattribute__(name)
+        try:
+            return super().__getattribute__(name)
+        except AttributeError:
+            if name.startswith("model_") or cls.__name__ in ("BaseInputs", "ExternalInputs"):
+                raise
+
+            if getattr(cls, "__descriptor_class__", None) is ExternalInputReference:
+                return ExternalInputReference(name=name, types=(), instance=None, inputs_class=cls)
+            else:
+                return WorkflowInputReference(name=name, types=(), instance=None, inputs_class=cls)
```

Comment:
I would just return this guy instead of doing the `super().__getattribute__(name)`